pkg/network/master/subnets.go

```diff
@@ -19,92 +19,67 @@ import (
 	"github.com/openshift/origin/pkg/network/common"
 )
 
-func (master *OsdnMaster) SubnetStartMaster(clusterNetworks []common.ClusterNetwork) error {
-	subrange := make(map[common.ClusterNetwork][]string)
-	subnets, err := master.hostSubnetInformer.Lister().List(labels.Everything())
```

Comment:
as with the vnid code, we have to keep the initial population-from-existing HostSubnets, because otherwise things will get messed up if we see the Node Added events before the corresponding HostSubnet Added events.